### PR TITLE
[cli] Symlink `@now/build-utils` when installing builders in "vc build"

### PR DESCRIPTION
**Pull request by TooTallNate**
To support legacy Builders like `@frontity/now` which still depends on the old `@now/build-utils` package name, create a symlink to `@vercel/build-utils` to make import the package work transparently.

Depends on #7842 being merged first.

### CODE REVIEW

File: packages/cli/test/unit/util/dev/server.test.ts [hunk 1]
```diff
@@ -1,438 +0,0 @@
...
-async function runNpmInstall(fixturePath: string) {
-  if (await fs.pathExists(path.join(fixturePath, 'package.json'))) {
-    return execa('yarn', ['install', '--network-timeout', '1000000'], {
-      cwd: fixturePath,
-      shell: true,
-    });
-  }
-}
-
-interface TestFixtureOptions extends Omit<DevServerOptions, 'output'> {
```
Comment (styfle) [inline packages/cli/test/unit/util/dev/server.test.ts]: Why was this deleted?

File: packages/cli/src/util/build/import-builders.ts [hunk 2]
```diff
@@ -203,6 +203,11 @@ async function installBuilders(
     cwd: buildersDir,
   });
 
+  // Symlink `@now/build-utils` -> `@vercel/build-utils` to support legacy Builders
+  const nowScopePath = join(buildersDir, 'node_modules/@now');
```
Comment (styfle) [inline packages/cli/src/util/build/import-builders.ts:207]:
```suggestion
  const nowScopePath = join(buildersDir, 'node_modules', '@now');
```

Comment (TooTallNate) [reply packages/cli/src/util/build/import-builders.ts:207]: There's no difference

File: packages/cli/test/unit/util/dev/server.test.ts (hunk 1 above)
Comment (TooTallNate) [reply packages/cli/test/unit/util/dev/server.test.ts]: Carry-over from #7845. Not included in the diff here anymore since that PR got merged.

**Review by styfle — APPROVED** [review]
